pallets/phala/src/registry.rs

```diff
@@ -294,6 +310,46 @@ pub mod pallet {
 			));
 			Ok(())
 		}
+
+		/// Register a PRuntime
+		#[pallet::weight(0)]
+		pub fn add_pruntime(
+			origin: OriginFor<T>,
+			pruntime_hash: PRuntimeHash,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let mut allowlist = PRuntimeAllowList::<T>::get().unwrap_or(Vec::new());
+			ensure!(!allowlist.contains(&pruntime_hash), Error::<T>::PRuntimeAlreadyExist);
+
+			allowlist.push(pruntime_hash);
+			PRuntimeAllowList::<T>::put(allowlist);
+
+			Ok(())
+		}
+
+		#[pallet::weight(0)]
+		pub fn remove_pruntime(
+			origin: OriginFor<T>,
+			pruntime_hash: PRuntimeHash,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let mut allowlist = PRuntimeAllowList::<T>::get().unwrap_or(Vec::new());
+			ensure!(allowlist.contains(&pruntime_hash), Error::<T>::PRuntimeNotFound);
+
+			let len = allowlist.len();
+			for i in 0..len {
+				if allowlist[i] == pruntime_hash {
+					allowlist.remove(i);
+					break;
+				}
+			}
```

Comment:
```suggestion
			let filtered: Vec<_> = allowlist.into_iter().filter(|h| h == pruntime_hash).collect();
```

Reply:
should be `let filtered: Vec<_> = allowlist.into_iter().filter(|h| *h != pruntime_hash).collect();` because we wanna remove that value